src/BF2TV.WindowsApp/Services/GameDataLookup/Battlefield2/KnownConfigSettings.cs

```diff
@@ -0,0 +1,25 @@
+﻿using BF2TV.WindowsApp.Services.GameDataLookup.Battlefield2.Files;
+
+namespace BF2TV.WindowsApp.Services.GameDataLookup.Battlefield2;
+
+public static class KnownConfigSettings
```

Comment:
- [ ] Refactor these methods into their respective config files.

Just makes more sense to have them there, even though I love how these extension methods have decoupled them selves from an earlier implementation.